src/fft.jl

```diff
@@ -1,63 +1,121 @@
 import FFTW
 
+# returns the lengths of the bounding rectangle in reciprocal space
+# that encloses the sphere of radius Gmax
```

Comment:
I know but I like indicating internal functions by their absence of docstrings (it distinguishes them when looking at the source)

Reply:
I see, but (a) we are far from consistent with that and (b) If you go through the effort of writing something, why not just make it accessible to the julia shell using the docstring system that has been created for exactly that purpose?

Reply:
I know you use the julia shell documenter, but I don't like it as much as you do because it makes the source code much more verbose. So I'd rather we keep the docstrings for external-facing functions, which as an added bonus makes it easier to see which functions you're "supposed" to use and which you're not. We can try to make it more consistent if that's your opposition. I don't have a strong position on the use of underscores

Reply:
Well but in this case it's 8 characters and two lines more: `"""\n` and `\n"""`. Sounds fair, no?

Reply:
Yeah but again it makes it less clear that this is a helper function that you're not supposed to use from the outside

Reply:
I'm OK with docstrings if we find another way to make that clear. I don't particularly like underscores but we could do that

Reply:
How about not `export`ing it?

Reply:
That doesn't help you when you're looking at the file. This information is far away in DFTK.jl

Reply:
Yes that's a valid point. Regarding the underscores thing the tipping point for me is how many functions this would affect. If it's many (like most in DFTK) than I would not do that. If it's occasional (like 2-3 as many underscore functions as we have now) I would agree to that solution.